app/controllers/static_pages_controller.rb

```diff
@@ -13,4 +15,10 @@ def terms_of_use; end
   def success_stories
     @success_stories = SuccessStory.all
   end
+
+  def import_content
```

Comment:
To keep everything within the admin files, we could use a [page_action](https://activeadmin.info/10-custom-pages.html#add-a-page-action) for this in the admin dashboard file:

```ruby
page_action :reseed, method: :post do
    github_paths = Lesson.pluck(:github_path)
    UpdateLessonContentJob.perform_async(github_paths)
    redirect_to admin_dashboard_path, notice: 'Curriculum lesson import job running 🐢 Track the progress at /sidekiq'
  end
```

Reply:
Was very close to going down this route, figured it was better to make it active admin agnostic but I do agree it's better to keep it separate. Shouldn't be too hard to change in the future in any case

Reply:
It's solid reasoning making it agnostic mate. We'll definitely have to retrace some steps when it comes to new admin by going back and redoing some of this stuff. But I think it'll be worth it just to have the boundaries clear between admin and the rest of the app until then.

Reply:
Yeah after doing it this way, I agree it feels much tidier / self contained :ok_hand: